spec/factories/user.rb

```diff
@@ -0,0 +1,15 @@
+FactoryGirl.define do
```

Comment:
We try to keep the factories in the same file, as suggested in https://github.com/thoughtbot/guides/tree/master/style/testing